diff --git a/samples/CMakeLists.txt b/samples/CMakeLists.txt
index 880f5709..4a0f0715 100644
--- a/samples/CMakeLists.txt
+++ b/samples/CMakeLists.txt
@@ -12,7 +12,7 @@ add_subdirectory(blend)
 add_subdirectory(foot_ik)
 add_subdirectory(look_at)
 add_subdirectory(millipede)
-add_subdirectory(multithread)
+#add_subdirectory(multithread)
 add_subdirectory(optimize)
 add_subdirectory(partial_blend)
 add_subdirectory(playback)
