diff --git a/contrib/Configure.cmake b/contrib/Configure.cmake
index b627855..394ec35 100644
--- a/contrib/Configure.cmake
+++ b/contrib/Configure.cmake
@@ -14,9 +14,12 @@ print_option_status (SOLOUD_BUILD_DEMOS "Build demos")
 option (SOLOUD_BACKEND_NULL "Set to ON for building NULL backend" ON)
 print_option_status (SOLOUD_BACKEND_NULL "NULL backend")
 
-option (SOLOUD_BACKEND_SDL2 "Set to ON for building SDL2 backend" ON)
+option (SOLOUD_BACKEND_SDL2 "Set to ON for building SDL2 backend" OFF)
 print_option_status (SOLOUD_BACKEND_SDL2 "SDL2 backend")
 
+option (SOLOUD_BACKEND_ALSA "Set to ON for building ALSA backend" OFF)
+print_option_status (SOLOUD_BACKEND_ALSA "ALSA backend")
+
 option (SOLOUD_BACKEND_COREAUDIO "Set to ON for building CoreAudio backend" OFF)
 print_option_status (SOLOUD_BACKEND_COREAUDIO "CoreAudio backend")
 
diff --git a/contrib/src.cmake b/contrib/src.cmake
index 2f9bcda..6e79481 100644
--- a/contrib/src.cmake
+++ b/contrib/src.cmake
@@ -166,6 +166,25 @@ if (SOLOUD_BACKEND_SDL2)
 
 endif()
 
+
+if (SOLOUD_BACKEND_ALSA)
+	#find_package (SDL2 REQUIRED)
+	include_directories (/usr/include)
+	add_definitions (-DWITH_ALSA)
+
+	set (BACKENDS_SOURCES
+		${BACKENDS_SOURCES}
+		${BACKENDS_PATH}/alsa/soloud_alsa.cpp
+	)
+
+	set (LINK_LIBRARIES
+		${LINK_LIBRARIES}
+		asound
+	)
+
+endif()
+
+
 if (SOLOUD_BACKEND_COREAUDIO)
 	if (NOT APPLE)
 		message (FATAL_ERROR "CoreAudio backend can be enabled only on Apple!")
